src/objective-c/tests/Podfile

```diff
@@ -39,6 +39,22 @@ target 'CoreCronetEnd2EndTests' do
   pod 'gRPC-Core/Cronet-Tests', :path => GRPC_LOCAL_SRC
 end
 
+target 'InteropTestsRemoteWithCronet' do
+  pod 'Protobuf', :path => "#{GRPC_LOCAL_SRC}/third_party/protobuf", :inhibit_warnings => true
+
+  pod '!ProtoCompiler',            :path => "#{GRPC_LOCAL_SRC}/src/objective-c"
+  pod '!ProtoCompiler-gRPCPlugin', :path => "#{GRPC_LOCAL_SRC}/src/objective-c"
+
+  pod 'BoringSSL',       :podspec => "#{GRPC_LOCAL_SRC}/src/objective-c", :inhibit_warnings => true
+  pod 'CronetFramework', :podspec => "#{GRPC_LOCAL_SRC}/src/objective-c"
+  
+  pod 'gRPC',           :path => GRPC_LOCAL_SRC
+  pod 'gRPC-Core',      :path => GRPC_LOCAL_SRC
+  pod 'gRPC-RxLibrary', :path => GRPC_LOCAL_SRC
+  pod 'gRPC-ProtoRPC',  :path => GRPC_LOCAL_SRC
+  pod 'RemoteTest', :path => "RemoteTestClient"
```

Comment:
Let's put this whole block into one or a few functions, instead of copy-pasting it, so it's not a pain to update. Something like:

```ruby
target  'InteropTestsRemoteWithCronet' do
  protoc_and_plugin
  grpc_runtime
  pod 'CronetFramework', :podspec ...
  ...
end
```